kernels/volk/volk_32f_index_min_16u.h

```diff
@@ -0,0 +1,375 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2021 Free Software Foundation, Inc.
```

Comment:
We switched to DCO, so you don't need to assign copyright to the FSF anymore. Furthermore, did you sign a CLA with the FSF in the past? If yes, you can still assign copyright if you want to, if no, you shouldn't.

Reply:
I signed the CLA two years ago (June 2019).